pkg/domain/infra/abi/play.go

```diff
@@ -722,6 +722,7 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 			RestartPolicy:      ctrRestartPolicy,
 			SeccompPaths:       seccompPaths,
 			SecretsManager:     secretsManager,
+			PidNSIsHost:        p.Pid.IsHost(),
```

Comment:
Can we put this in alpha order here, i.e. move this line to 716.5?

Reply:
Sorry, I completly forgot about that, fixing in https://github.com/containers/podman/pull/17186